### PR TITLE
Filenames

### PR DESCRIPTION
**Pull request by jacopo-exact**

### CODE REVIEW

File: fractal_server/app/runner/_slurm/executor.py [hunk 1]
```diff
@@ -57,14 +66,20 @@ def __init__(
         self.script_dir: Path = script_dir  # type: ignore
 
     def get_stdout_filename(self, arg: str = "%j") -> Path:
-        return self.script_dir / f"slurmpy.stdout.{arg}.log"
+        return self.script_dir / f"slurmpy.stdout.{arg}.out"
+
+    def get_stderr_filename(self, arg: str = "%j") -> Path:
```
Comment (tcompa) [inline fractal_server/app/runner/_slurm/executor.py:71]: ?

File: fractal_server/app/runner/_slurm/executor.py [hunk 2]
```diff
@@ -192,9 +210,18 @@ def _result_or_cancel(fut, timeout=None):
...
         fs = [
             self.submit(
-                fn, *args, additional_setup_lines=additional_setup_lines
+                fn,
+                *args,
+                additional_setup_lines=additional_setup_lines,
+                job_stdout=job_file_fmt.format(
+                    args=sanitize_string(args[0]), suffix=".out"
+                ),
+                job_stderr=None,
```
Comment (tcompa) [inline fractal_server/app/runner/_slurm/executor.py:224]: typo?

File: fractal_server/app/runner/_slurm/executor.py [hunk 3]
```diff
@@ -236,15 +266,16 @@ def submit(
         job = SlurmJob()
         job.slurm_input = self.get_in_filename(job.workerid)
         job.slurm_output = self.get_out_filename(job.workerid)
+        job.slurm_script = self.get_slurm_script_filename(random_string())
```
Comment (tcompa) [inline fractal_server/app/runner/_slurm/executor.py:269]: Should there be also `job.slurm_error`?

File: tests/test_backend_slurm.py [hunk 4]
```diff
@@ -182,3 +182,16 @@ def test_sbatch_script_slurm_config(
         )
         assert job_name
         assert len(job_name.split()[-1]) == len(task.name)
+
+        sbatch_script = "".join(sbatch_script_lines)
+        debug(sbatch_script)
+        if "task_parallel" in sbatch_script:
+            output_line = next(
```
Comment (tcompa) [inline tests/test_backend_slurm.py:189]: Also test `error_line`?

File: fractal_server/app/runner/_slurm/executor.py (hunk 1 above)
Comment (jacopo-exact) [reply fractal_server/app/runner/_slurm/executor.py:71]: just default values

File: fractal_server/app/runner/_slurm/executor.py (hunk 3 above)
Comment (jacopo-exact) [reply fractal_server/app/runner/_slurm/executor.py:269]: no, these are the pickled files.

**Pull request merged**